mllib/src/main/scala/org/apache/spark/mllib/classification/NaiveBayes.scala

```diff
@@ -446,8 +460,14 @@ object NaiveBayes {
   /** String name for Bernoulli model type. */
   private[spark] val Bernoulli: String = "bernoulli"
 
+  /** String name for complementary multinomial model type. */
+  private[spark] val Complementary_Multinomial: String = "complementary_multinomial"
```

Comment:
No, but you can try to refactor the existing implementations to have some "complementary" config. It's possible that later these are exposed as separate types, but underneath, no need to be.